yarn-project/circuit-types/src/packed_values.ts

```diff
@@ -1,31 +1,22 @@
 import { Fr, Vector } from '@aztec/circuits.js';
 import { computeVarArgsHash } from '@aztec/circuits.js/hash';
 import { BufferReader, serializeToBuffer } from '@aztec/foundation/serialize';
-import { type FieldsOf } from '@aztec/foundation/types';
 
 /**
  * Packs a set of values into a hash.
  */
 export class PackedValues {
```

Comment:
Cleaned this up a bit to make it obvious that it's immutable and that the values pass in via `fromValues`.